webservices/legal_docs/current_cases.py

```diff
@@ -20,7 +19,7 @@
 logger = logging.getLogger(__name__)
 
 # for debug, uncomment this line
-# logger.setLevel(logging.DEBUG)
+logger.setLevel(logging.DEBUG)
```

Comment:
Please comment out this debug line. Thanks

Reply:
@fec-jli debug statement is now commented and the changes are pushed to this branch.